packages/components/typography/src/heading.tsx

```diff
@@ -0,0 +1,25 @@
+import styled from "styled-components";
+import { Text, TextProps } from "./text";
+
+export type HeadingSize = "s" | "m" | "l" | "xl" | "2xl" | "3xl";
+
+interface OwnProps {
+  size: HeadingSize;
+}
+
+type HeadingProps = TextProps & OwnProps;
+
+export const HEADING_TOKEN_MAPPING = {
```

Comment:
Is it worth typing these maps with something like `Record<HeadingSize, keyof typeof theme.fontSizes>`

Reply:
Not a bad idea